<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionScopeEvaluation</class>
 <widget class="QWidget" name="MotionScopeEvaluation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1188</x>
     <y>30</y>
     <width>488</width>
     <height>784</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{background: #FFFFFF;
border-radius: 12px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="train_GroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>448</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#train_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#train_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 5px;}</string>
    </property>
    <property name="title">
     <string>评定设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="fourFinger_Btn">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>45</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/boostFourNoSelected.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/boostFourHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="oneFinger_Btn">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>45</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image: url(:/new/prefix1/source/buttons/level/boostOneNoSelected.png);}
QPushButton:hover{border-image: url(:/new/prefix1/source/buttons/level/boostOneHover.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="stretch_GroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>242</y>
      <width>190</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#stretch_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#stretch_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 1px;}</string>
    </property>
    <property name="title">
     <string>伸展阻力</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>37</y>
       <width>88</width>
       <height>76</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/source/dividingRule/speedDial.png);</string>
     </property>
     <widget class="QDial" name="stretch_Dial">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>9</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="buckling_GroupBox">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>242</y>
      <width>190</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#buckling_GroupBox{border-radius: 8px;
border: 1px solid #979797;
margin-top:4ex;}
#buckling_GroupBox::title{subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 1px;}</string>
    </property>
    <property name="title">
     <string>屈曲阻力</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>37</y>
       <width>88</width>
       <height>76</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/source/dividingRule/speedDial.png);</string>
     </property>
     <widget class="QDial" name="buckking_Dial">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>9</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#333333;</string>
   </property>
   <property name="text">
    <string>关节活动度评定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>916</x>
     <y>232</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中指</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1104</x>
     <y>278</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="thumb_Label">
   <property name="geometry">
    <rect>
     <x>984</x>
     <y>130</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="littleFinger_Label">
   <property name="geometry">
    <rect>
     <x>984</x>
     <y>309</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1104</x>
     <y>190</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>1104</x>
     <y>322</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="middleFinger_Label">
   <property name="geometry">
    <rect>
     <x>984</x>
     <y>220</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>916</x>
     <y>187</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>食指</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1104</x>
     <y>232</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>916</x>
     <y>277</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>无名指</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>914</x>
     <y>321</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小指</string>
   </property>
  </widget>
  <widget class="QLabel" name="forefinger_Label">
   <property name="geometry">
    <rect>
     <x>984</x>
     <y>175</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>916</x>
     <y>142</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拇指</string>
   </property>
  </widget>
  <widget class="QLabel" name="ringFinger_Label">
   <property name="geometry">
    <rect>
     <x>984</x>
     <y>265</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1104</x>
     <y>142</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>753</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>829</x>
     <y>750</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总关节度的</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="totalPercent_Label">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>740</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFFFFF;
border: 1px solid #0D9DDB;</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_Btn">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>844</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 4px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 4px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="CTrainingHand" name="train_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>111</y>
     <width>581</width>
     <height>585</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reEvaluat_Btn">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>844</y>
     <width>164</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: #0D9DDB;border-radius: 4px;color:white;}
QPushButton:hover{background: #1EAACF;border-radius: 4px;color:#333333;}</string>
   </property>
   <property name="text">
    <string>评估</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTrainingHand</class>
   <extends>QWidget</extends>
   <header>ctraininghand.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
